meshtastic/interfaces/ble/compatibility_service.py

```
@@ -0,0 +1,377 @@
+        thread_drain = getattr(thread, "_drain_publish_queue", None)
+        if callable(thread_drain) and not _is_unconfigured_mock_callable(thread_drain):
+            try:
+                BLECompatibilityEventService._safe_execute(
+                    iface,
+                    lambda: thread_drain(flush_event),
+                    error_msg="Error draining publish queue via publishing thread",
+                    reraise=True,
+                )
+                return
+            except Exception:  # noqa: BLE001 - fall back to direct draining path
+                pass
```

Ruff (0.15.5)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)